src/client/dfuse/ops/rename.c

```diff
@@ -70,13 +70,25 @@ dfuse_cb_rename(fuse_req_t req, struct dfuse_inode_entry *parent,
 
 	fs_handle = fuse_req_userdata(req);
 
-	if (flags != 0)
+	if (flags != 0) {
+#ifdef RENAME_NOREPLACE
+		if (flags != RENAME_NOREPLACE) {
+			if (flags & RENAME_EXCHANGE)
+				DFUSE_TRA_DEBUG(parent, "Unsupported flag RENAME_EXCHANGE");
```

Comment:
we do have dfs_exchange() although not very well tested

Reply:
I can add that next, I've not seen any use-cases for this in the wild.

Reply:
indeed

Reply:
I wasn't, so thanks for the heads-up.